### ISSUE TITLE
HRPD: McLennan rotation stage need to be setup

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#3664]
As an HRPD instrument scientist I need help to setup a mclennan rotation stage on Monday at 9am.

**Comment by Tom-Willemsen** [comment]
```
ACCL1=1
AXIS1=yes
CMOD1=CLOSED
DHLM1=360
DLLM1=-360
ERES1=1/1
HOME1=1
MSTP1=5.5555555555
MTRCTRL=1
NAME1=HRPD
UNIT1=deg
VELO1=1
```

**Comment by John-Holt-Tessella** [comment]
All done. Extra information:

1. I generated a short script for them to run the experiment
1. Homing mode needed to be 0 (Hardware defined) It should be homing to the datum not the reverse limit
1. With homing mode set to 0 (as with 1 and 2) it does not set the dial and user coordinates to 0. I have created a script for them which does this that they should run after a home. it is `inst.zero_mclennan_after_home`. If the motor is homed forward it might not hit the datum in which case it hits the limit (and presumably the same for reverse), if you are doing this or asked about it make sure the reading makes sense before zeroing.
